File: java/Shirt.java
public class Shirt extends Clothing {
    private String color;
    private String sleeveType;

    // Default constructor
    public Shirt() {
        super(); // Calling superclass constructor
        this.color = "";
        this.sleeveType = "";
    }

    // Parameterized constructor
    public Shirt(String size, String material, String gender, String color, String sleeveType) {
        super(size, material, gender); // Calling superclass constructor
        this.color = color;
        this.sleeveType = sleeveType;
    }

    // Getters
    public String getColor() {
        return color;
    }

    public String getSleeveType() {
        return sleeveType;
    }

    // Setters
    public void setColor(String color) {
        this.color = color;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }
}

File: python/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    def __init__(self, size="", material="", gender="", color="", sleeve_type=""):
        super().__init__(size, material, gender)
        self.__color = color
        self.__sleeve_type = sleeve_type

    def get_color(self):
        return self.__color

    def set_color(self, color):
        self.__color = color

    def get_sleeve_type(self):
        return self.__sleeve_type

    def set_sleeve_type(self, sleeve_type):
        self.__sleeve_type = sleeve_type
